libs/detectors/jetson/openpifpaf_tensorrt/openpifpaf_tensorrt.py

```diff
@@ -0,0 +1,212 @@
+import time
```

Comment:
Is this file a copy of the x86 version?

Reply:
x86? no but yes for x86-gpu-tensorrt
This happens a lot on other places too due to the policy to separate modules based on platforms to have a straightforward structure. There is a solution to change the structure and add one gpu-tensorrt device for both x86-gpu-tensorrt and Jetson devices.